### ISSUE TITLE
Obsidian

### ISSUE THREAD

**Issue by SimonLab** [dwyl/product-ux-research#23]
[![image](https://user-images.githubusercontent.com/6057298/98121221-f04e4d80-1ea6-11eb-82ed-037dda4d3e39.png)](https://obsidian.md/)

![image](https://user-images.githubusercontent.com/6057298/98121824-b7fb3f00-1ea7-11eb-879b-d9de7d31c6ff.png)

- https://news.ycombinator.com/item?id=23324598:
![image](https://user-images.githubusercontent.com/6057298/98122710-e0376d80-1ea8-11eb-90e0-6fa5850fd70b.png)

**Comment by nelsonic** [comment]
@SimonLab thanks for opening this. 👌 
https://obsidian.md/pricing
![image](https://user-images.githubusercontent.com/194400/218920092-b4665a94-2689-49de-8a07-f21a4dea6af2.png)

"Free" for use on `localhost`. but if you want to sync between your devices, **`$8/month`**. 

None of their _actual_ code is Open Source
https://github.com/orgs/obsidianmd/repositories
<img width="1091" alt="image" src="https://user-images.githubusercontent.com/194400/218919636-9a03f9df-9dec-49a8-b5d8-49bb256b1f61.png">

https://github.com/obsidianmd/obsidian-releases 
<img width="1277" alt="image" src="https://user-images.githubusercontent.com/194400/218919831-26658dbb-540d-4297-9d0b-26bd448530d8.png">

**Comment by SimonLab** [comment]
Because obsidian save notes in "vaults" which are just folders, I was sync between multiple devices using Github.
You  have to know how to use git and push/pull the contents so there are a few steps to make this work. I think it's possible to make it work also on mobile too but I haven't tried.

**Comment by nelsonic** [comment]
Yeah, manual git-sync is _waaaayyy_ too much "work" for a non-technical person 👎 
or even a technical person who is even _slightly_ busy (or lazy). ⏳ 
Sync should be automatic. Backups should be automatic. 🪄 

Very curious to know what parts of your life/work you've used Obsidian for. 💭 
Guessing you didn't use it for anything _collaborative_ with your friends, family, colleagues? 🤔

**Comment by SimonLab** [comment]
I'm using Obsidian sporadically to takes notes.
If I'm on my phone, I'm using google keep to take quick notes that I can review and convert to obsidian later on if necessary or I just delete them.

I don't use Obsidian as a collaborative tool.

**Comment by nelsonic** [comment]
Makes sense. thanks for sharing. 👌
